Assets/HoloToolkit-Examples/SpatialUnderstanding/SpatialUnderstanding-FeatureOverview/Scripts/AppState.cs

```diff
@@ -1,11 +1,14 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using UnityEngine;
 using HoloToolkit.Unity;
-using System;
 using HoloToolkit.Unity.InputModule;
 using HoloToolkit.Unity.SpatialMapping;
+using System;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
I'm only using it for a ToArray() call, which doesn't involve any more allocations than we actually need.  I suspect the performance guidance is saying to avoid chains of select/where/groupby/etc, since those would involve many GC allocations.  I'll see if I can get that guidance clarified.

Are people concerned that by adding the System.Linq using, I'm encouraging people to use Linq freely?  If so, I can Explicitly call System.Linq.Enumerable.ToArray() on the object instead.